askar-storage/src/options.rs

```diff
@@ -126,7 +136,9 @@ fn percent_encode_into(result: &mut String, s: &str) {
     push_iter_str(result, utf8_percent_encode(s, NON_ALPHANUMERIC))
 }
 
+/// A trait implemented by types that can be converted into Options
 pub trait IntoOptions<'a> {
```

Comment:
Is there a reason why `TryInto` for Options is not used? If this for some lower Rust version compatibility?

Reply:
It's actually because that creates issues accepting a straight Options instance - it implements `From<Self>` which means `TryFrom<Self>` is implemented with the `Infallable` error type, not the crate's error type.